css/css-flexbox/reference/align-content_space-around-ref.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>CSS Flexible Box Test: align-content_space-around</title>
+<link rel="stylesheet" href="../support/test-style.css">
+<style>
+#test {
+  height: 200px;
+  width: 80px;
+  align-content: center;
+}
+#spacertop {
+  width: 50px;
+  height: 8px;
+}
+#spacermiddle {
+  width: 50px;
+  height: 17px;
```

Comment:
Hmm, this does not seem to work well with subpixel layout, but since you say the tests pass I guess it's fine. But ideally the test should be rewritten so that things divide to an integer.

Reply:
Great idea! I would change the height to 210 in this case so numbers are perfectly round.

Reply:
Sounds good!